client/index.jsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+ReactDOM.render(<div>Hello world</div>, document.getElementById('app'));
```

Comment:
@MarissaShieh Please add a newline break at the end of the file to meet style guide.